document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/document/DocumentGenerator.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.companieshouse.document.generator.core.document;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.companieshouse.document.generator.core.avro.AvroDeserializer;
+import uk.gov.companieshouse.document.generator.core.document.models.RenderSubmittedDataDocument;
+import uk.gov.companieshouse.document.generator.core.kafka.ConsumerGroupHandler;
+import uk.gov.companieshouse.document.generator.core.kafka.configurationhelper.KafkaConfigHelper;
+import uk.gov.companieshouse.document.generator.interfaces.DocumentInfoService;
+import uk.gov.companieshouse.document.generator.interfaces.model.DocumentInfo;
+import uk.gov.companieshouse.kafka.consumer.CHKafkaConsumerGroup;
+import uk.gov.companieshouse.kafka.message.Message;
+
+import java.util.Arrays;
+
+public class DocumentGenerator implements Runnable {
+
+    private static final String[] CONSUMER_TOPICS = new String[]{
```

Comment:
Any reason we have a String array here? and not a list? That would save us having to convert to a list when we use it?